lib/vue-lib/src/pinia/pinia_api_tools.ts

```diff
@@ -462,6 +461,8 @@ export const initPiniaApiToolkitPlugin = (config: { api: AxiosInstance }) => {
           completed.resolve({
             error: err,
           });
+          span.setAttributes({ "http.status_code": err.response.status });
+          span.end();
```

Comment:
Ah, was ending the span a necessary part of the solution? I guess that makes a lot of sense, there isn't something like a Rust `Drop` trait to fall back on.

Reply:
Yep